mantidproject/mantid
# Fix summation error on instrument view and ALF view

**Description of work.**
This PR fixes an error where the values in a Y histogram are not being summed correctly because the starting number is 0 as an integer rather than a double.

**To test:**
Select ALF instrument in settings, and turn on data archive.
Open ALFView
Load run 83743
Click on Pick tab.
Select a rectangular region on the InstrumentWidget
The RHS plot should look like this:
![image](https://user-images.githubusercontent.com/40830825/206154003-63d968f9-cc6b-46af-aeb2-2e727f404915.png)



Fixes #34815





---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Are the release notes saved in a separate file, using Issue or PR number for file name and in the correct location?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
